Woof/Woof/Model/PetOwner.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/**
+ A struct representing a pet owner.
+ This struct conforms to the Identifiable and Codable protocols, allowing it
+ to be easily used in SwiftUI and encoded/decoded from JSON.
+ */
```

Comment:
Your documentation does not give any new information:
```suggestion
/**
A person who owns a pet (or several pets) and might be interested in finding a pet sitter for them.
 */
```

Reply:
Ok, your suggestion looks clear. I'll change it.